src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -166,6 +166,7 @@ public partial class CellEditorComponent :
 
     private readonly List<EditorUserOverride> ignoredEditorWarnings = new();
 
+    private readonly Compound atp = SimulationParameters.Instance.GetCompound("atp");
```

Comment:
This should be outside the pragma warning disable block.

Reply:
It is

Reply:
But line 170 shows in the diff as "#pragma warning disable CA2213" to me which means that this *is* inside the pragma block.

Reply:
I thought it's like:

> disable

disabled content

> restore

Reply:
No wait, you are correct. I'm just misreading everything this morning apparently. Could you add a blank line before that starting pragma? That'll make it visually clear to me that this line is not inside the pragma block and it is before it.